src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -164,6 +165,9 @@ private void Initialize(AnalyzerExecutor analyzerExecutor, DiagnosticQueue diagn
                 newOnAnalyzerException = (ex, analyzer, diagnostic) => addDiagnostic(diagnostic);
             }
 
+            // Assume all analyzers are non-thread safe.
```

Comment:
They were touched by the original PR, so they got reset to their original state.  I didn't notice they were unused.  Will remove.